cmd/registry/patch/version.go

```diff
@@ -129,13 +131,13 @@ func applyApiVersionPatch(
 		return fmt.Errorf("UpdateApiVersion: %s", err)
 	}
 	for _, specPatch := range version.Data.ApiSpecs {
-		err := applyApiSpecPatch(ctx, client, specPatch, name.String(), "")
```

Comment:
This was the problem, which @seaneganx recently observed was breaking spec uploads from YAML.